controllers/templatesync/template_sync.go

```diff
@@ -1249,7 +1249,7 @@ func overrideRemediationAction(instance *policiesv1.Policy, tObjectUnstructured
 func (r *PolicyReconciler) emitTemplateSuccess(
 	ctx context.Context, pol *policiesv1.Policy, tIndex int, tName string, clusterScoped bool, msg string,
 ) error {
-	err := r.emitTemplateEvent(ctx, pol, tIndex, tName, clusterScoped, "Normal", "Compliant; ", msg)
+	err := r.emitTemplateEvent(ctx, pol, tIndex, tName, clusterScoped, "Normal", string(policiesv1.Compliant), msg)
```

Comment:
Instead, could you please change `emitTemplateEvent` to accept a compliance state argument instead of `msgMeta`?  It seems only in the case of `emitTemplateError` does it add an additional prefix, so this can just be added to the message instead.